Sources/Kitura/CodableRouter.swift

```diff
@@ -19,6 +19,11 @@ import LoggerAPI
 import KituraNet
 import KituraContracts
 
+/// Bridge [RequestError](https://ibm-swift.github.io/KituraContracts/Structs/RequestError.html)
```

Comment:
As far as I can tell, this dummy class was only necessary because the file otherwise contains only a `typealias`.  In this case I dropped the typealias into the existing `CodableRouter.swift` since I could argue it belongs there.

Reply:
OK as long as Jazzy is generating the typealias OK then we're fine :)

Reply:
Tested locally, it generates fine.